helm/ffc-demo-payment-web/values.yaml

```diff
@@ -26,6 +26,8 @@ container:
   allowPrivilegeEscalation: false
   restartPolicy: Always
   staticCacheTimeoutMillis: 54000
+  restClientTimeoutMillis: 20000
+  paymentServiceUrl: http://ffc-demo-payment-service.ffc-demo
```

Comment:
This doesn't look right given the line above:

 `PAYMENT_SERVICE_URL: http://ffc-demo-payment-service:3004`

Is it because the value is more the service name rather the paymentServiceUrl?

Reply:
this is correct for an address in the cluster, which is `{service-name}.{namespace-name}`, with the service being mapped to port 80 within k8s rather than 3004.

For other repos we have the default as the service address in the 'standard' namespace of 'ffc-demo', so this follows that convention.